### PR TITLE
Use value as the operand Spec

### PR DESCRIPTION
**Pull request by horis233**
**What this PR does / why we need it**:

This is for avoiding the `null` spec in the operandrequest

Screen Shot 2020-12-28 at 10 50 31 PM

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

File: api/v1alpha1/zz_generated.deepcopy.go [hunk 1]
```diff
@@ -129,7 +129,11 @@ func (in *Operand) DeepCopyInto(out *Operand) {
 			(*out)[key] = val
 		}
 	}
-	in.Spec.DeepCopyInto(&out.Spec)
+	if in.Spec != nil {
+		in, out := &in.Spec, &out.Spec
+		*out = new(runtime.RawExtension)
+		(*in).DeepCopyInto(*out)
+	}
```
Comment (Daniel-Fan) [inline api/v1alpha1/zz_generated.deepcopy.go:136]: May I ask that could we use func (in *RawExtension) DeepCopy() *RawExtension{} directly?

https://github.com/kubernetes/apimachinery/blob/15c5dba13c59f42b25aba4e3a6b17061af770216/pkg/runtime/zz_generated.deepcopy.go#L38

Comment (horis233) [reply api/v1alpha1/zz_generated.deepcopy.go:136]: This file is generated by `controller-gen`, which can't be overwritten by developers.

Comment (Daniel-Fan) [reply api/v1alpha1/zz_generated.deepcopy.go:136]: Oh! I see.

**Review by Daniel-Fan — APPROVED** [review]
/lgtm

**Pull request merged**